### ISSUE TITLE
import/newline-after-import doesn't warn comments

### ISSUE THREAD

**Issue by lgraziani2712** [benmosher/eslint-plugin-import#981]
Hello!

This specific code doesn't warn about the need of a new line after import:

```js
import { NamedExport } from 'path/to/module';
/**
 * Doc bla bla
 */
const defaultExport = () => ('bla bla');
```

I expect to get a warn about the import not having a new line. Is this behaviour correct? By the way, if I delete the comment the warn appears.

Thank you!
